### PR TITLE
Add gateways

### PR DESCRIPTION
**Pull request by thedeex**
Add DEEX gateways integration

### CODE REVIEW

File: app/lib/common/gateways.js [hunk 1]
```diff
@@ -189,6 +190,20 @@ export const availableGateways = {
...
         landing: "https://citadel.li/",
         wallet: "https://citadel.li/wallet/"
+    },
+    DEEX: {
+        id: "DEEX",
+        name: "DEEX",
+        baseAPI: deexAPIs,
+        isSimple: true,
+        simpleAssetGateway: false,
+        fixedMemo: {prepend: "dex:", append: ""},
```
Comment (serdrdoor) [inline app/lib/common/gateways.js]: Please see: fixedMemo in other integrations (RuDEX):
( "prepend_default" for DepositModal and [gateway].DepositRequest )

Comment (thedeex) [reply app/lib/common/gateways.js]: change prepend on prepend_default

File: app/lib/common/gateways.js [hunk 2]
```diff
@@ -189,6 +190,23 @@ export const availableGateways = {
...
+        fixedMemo: {
+            prepend_default: "dex:",
+            append: ""
+        },
+        isEnabled: _isEnabled("DEEX"),
+        addressValidatorMethod: "POST",
+        options: {
+            enabled: false,
+            selected: false
+        }
```
Comment (sschiessl-bcp) [inline app/lib/common/gateways.js]: Please add proper values for landing and wallet (see other gateways)

File: app/lib/common/gateways.js [hunk 3]
```diff
@@ -206,7 +206,9 @@ export const availableGateways = {
         options: {
             enabled: false,
             selected: false
-        }
+        },
+        landing: "https://deex.one/",
+        wallet: "https://deex.exchnage/"
```
Comment (sschiessl-bcp) [inline app/lib/common/gateways.js]: lol ...

Comment (thedeex) [reply app/lib/common/gateways.js]: :(

**Pull request merged**